src/main/scala/codacy/codesniffer/docsgen/parsers/PHPCSSecurityAuditDocsParser.scala

```diff
@@ -0,0 +1,31 @@
+package codacy.codesniffer.docsgen.parsers
+
+import better.files.File
+import codacy.codesniffer.docsgen.VersionsHelper
+import com.codacy.plugins.api.results.Pattern
+
+import scala.util.matching.Regex
+
+class PHPCSSecurityAuditDocsParser extends DocsParser {
```

Comment:
You should override the `fallbackCategory` from the superclass to `Security`.
https://github.com/codacy/codacy-codesniffer/blob/fe448a1a0571ad37dda34605d20fd21e11340c00/src/main/scala/codacy/codesniffer/docsgen/parsers/DocsParser.scala#L29